src/components/Header/Header.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Header extends React.Component {
+  state = {
+    value: '',
+  };
+
+  handleAddTodo = (event) => {
+    if (event.key === 'Enter' && event.target.value) {
+      this.setState((prevState) => {
+        this.props.onAddTodo(prevState.value);
+
+        return ({
+          value: '',
+        });
```

Comment:
```suggestion
        return {
          value: '',
        };
```
зачем тебе эти скобки? нужно везде убрать